src/API/Router.ts

```diff
@@ -91,16 +82,8 @@ export class Router {
   }
 
   private getWorks = async (context: KoaRouter.IRouterContext, next: () => Promise<any>) => {
-    this.logger.trace({ query: context.query }, 'GET /works')
-
-    const works = await this.workController.getWorks()
-
-    context.body = works
-  }
-
-  private getWorksByPublicKey = async (context: KoaRouter.IRouterContext, next: () => Promise<any>) => {
     this.logger.trace({ query: context.query }, '/works?publicKey=')
-    const publicKey = context.query.publicKey
+    const publicKey = context.query.publicKey ? context.query.publicKey : undefined
```

Comment:
I don't think this variable is necessary?  context.query.publickKey will either exist or not so it can be passed directly in the next line. There is the question of what happens if there's no context.query?

Using TDD will help with finding these kinds of edge cases as the code is developed.